airbyte-integrations/connectors/source-fastbill/source_fastbill/manifest.yaml

```diff
@@ -0,0 +1,116 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["RESPONSE","{{ parameters.record_extractor }}"]
+  requester:
+    type: HttpRequester
+    url_base: "https://my.fastbill.com/api/1.0"
+    http_method: "POST"
+    authenticator:
+      class_name: source_fastbill.components.CustomAuthenticator
+      username: "config['username']"
+      password: "config['api_key']"
+    request_body_json:
+      SERVICE: "{{ parameters.endpoint }}.get"
+      LIMIT: 100
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: "DefaultPaginator"
+      pagination_strategy:
+        type: "OffsetIncrement"
+        page_size: 100
+      page_token_option:
+        type: "RequestOption"
+        field_name: "OFFSET"
+        inject_into: "body_json"
+    requester:
+      $ref: "#/definitions/requester"
+  base_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/retriever"
+
+  invoices_stream:
+    $ref: "#/definitions/base_stream"
+    name: "invoices"
+    primary_key: "INVOICE_ID"
+    $parameters:
+      path: "/api.php"
```

Comment:
Maybe move the path to the base url. All streams use the same path.

Reply:
@marcosmarxm At first I have done that only but path can't be empty that's why I have to add this in path

Reply:
`path: ""` KEEPING PATH LIKE THIS ALSO NOT WORKING

Reply:
I see, thanks! Let's keep that as is